stanford_profile.profile

```diff
@@ -15,6 +16,19 @@ function stanford_profile_install_tasks(&$install_state) {
   return ['stanford_profile_final_task' => []];
 }
 
+/**
+ * Replace None option label with News for layout selection on Stanford News content type
```

Comment:
Mike and I reviewed this during our 1:1 -- this hook will need to be moved into `stanford_profile_helper/modules/stanford_news/stanford_news.module`

Reply:
For future Rebecca (this afternoon) look at: https://github.com/SU-SWS/stanford_profile_helper/pull/440